src/components/common/ImageUploader.tsx

```diff
@@ -55,16 +55,21 @@ const StyledSpinBlock = styled.div`
   transition: all 0.6s;
   background: rgba(0, 0, 0, 0.7);
 `
+type BasicAccept = 'image/*'
+type ExtraAccept = 'video/*' | 'audio/*' | 'application/*' | 'application/pdf' | 'text/plain' | 'application/zip'
 
 const ImageUploader: React.FC<{
   file: File | null
   initialCoverUrl?: string | null
   uploading?: boolean
   onChange?: (file: File) => void
   customName?: string
-}> = ({ file, initialCoverUrl, uploading, customName, onChange }) => {
+  extraAcceptType?: ExtraAccept
+}> = ({ file, initialCoverUrl, uploading, customName, extraAcceptType, onChange }) => {
   const { formatMessage } = useIntl()
   const [imgSrc, setImgSrc] = useState<string | null>(initialCoverUrl || null)
+  const basicAccept: BasicAccept = 'image/*'
+  const acceptType = extraAcceptType ? `${basicAccept},${extraAcceptType}` : basicAccept
```

Comment:
If it is written like this, does it mean that it can only accept two formats at most?
Maybe it can be more flexible.
For example, if it does not `accept` props, the default is `image/*`, but if it has `accepted` props, it will use props.